src/Symfony/Component/Cache/Adapter/FilesystemAdapter.php

```diff
@@ -148,10 +145,15 @@ protected function doSave(array $values, $lifetime)
         return $ok;
     }
 
-    private function getFile($id)
+    private function getFile($id, $mkdir)
```

Comment:
As `false` is used everywhere except in one call, what do you think of making it the default value?